app/views/art_items/_form.html.erb

```diff
@@ -66,7 +66,7 @@
   </div>
 
   <div id="attached_files" class="my-5">
-    <%= form.label :documents, "Documents (pdf,txt,jpg,png,doc)" %><span class="optional-text"> * optional</span>
+    <%= form.label :documents %><span class="optional-text">(5 Mb limit per item) * optional</span>
```

Comment:
I think it is 10 MB for documents.

Reply:
I changed it to 5MB for either image or doc. I reviewed what was in the store already and the largest document was 3.8MB